macros/answers/PGstringevaluators.pl

```diff
@@ -13,18 +14,14 @@ =head1 DESCRIPTION
 
 String answer evaluators compare a student string to the correct string.
 
-=head2 MathObjects and answer evaluators
-
-The MathObjects system provides a String->cmp() method that produce answer
-evaluators for string comparisons. It is recommended that you use the String
-object's cmp() method directly if possible.
-
```

Comment:
This shouldn't be removed.  It is basically saying not to use this macro when you have a MathObject `String` which is exactly what we want to encourage.